ui/src/app/pages/appAnalysis/app-files-list.tsx

```diff
@@ -113,12 +117,20 @@ export const AppFilesList = withCurrentWorkspace()(
       } = props;
       const { name } = row;
       const url = `${analysisTabPath(namespace, id)}/preview/${name}`;
-      return (
-        <Clickable>
```

Comment:
does removing this Clickable do anything?

Reply:
![Screenshot 2024-01-16 at 1 35 10 PM](https://github.com/all-of-us/workbench/assets/24514630/fcc535d5-b83b-40f8-b97f-e6d91b5dae85)
The only difference that I found was that for Clickable is that when it is there, the user sees the pointer icon when in the orange area instead of the default icon (when Clickable is not there). However, in both cases, clicking in the orange area appears to do nothing. Only clicking on the text will result in navigation.

Reply:
Thanks!  Good removal.